server/startup/migrations/v224.js

```diff
@@ -0,0 +1,10 @@
+import { Migrations } from '../../../app/migrations';
+import { Settings } from '../../../app/models';
+
+Migrations.add({
+	version: 224,
+	up() {
+		Settings.remove({ _id: 'UTF8_Names_Validation' });
```

Comment:
the migration should not remove the old setting, instead it should carry the **value** from the old setting as the value for the new ones.

imagine that a running rocket.chat server has customized the value of the old setting `UTF8_Names_Validation`, if they updated to a new version, they would lose the customized value and start using the new default values from the new settings. I'd say the RC admin would not be happy if that happens =)

Reply:
I took the value of the previous setting (`UTF8_Names_Validation`) and put that value in `UTF8_Channel_Names_Validation` and `UTF8_User_Names_Validation` in **migration 226**.